spec/components/input_spec.rb

```diff
@@ -215,13 +215,23 @@ def component_name
     assert_select ".govuk-label.govuk-label--xl"
   end
 
+  it "renders text input with prefix" do
+    render_component(
+      label: { text: "Cost per item, in pounds" },
+      name: "cost-per-item",
+      prefix: "£",
+    )
+    assert_select ".gem-c-input__wrapper .gem-c-input[name=cost-per-item]"
+    assert_select ".gem-c-input__wrapper .gem-c-input__affix[aria-hidden=true]", text: "£"
```

Comment:
Should this test check for the `--prefix` class? (same for line 235 but `--suffix`)

Reply:
Good shout, changed as recommended